js/App.js

```diff
@@ -125,11 +127,33 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits) {
+  if (maxDigits === undefined) { // Allow maxDigits = 0
```

Comment:
ah, i see. I suppose then you could do:

`maxDigits = maxDigits === undefined ? 8 : maxDigits`

but I guess it's a matter of personal preference.

Actually, even simpler you could take advantage of ECMAScript 6's default parameters and set the default right in the signature:

```
function(numberToFormat, maxDigits = 8)
```